### PR TITLE
fix for #2446 - removed maxDate

### PR DESCRIPTION
**Pull request by aks681**
I removed the condition that disabled the calendar based on a maxDate though the maxDate variable is still present for future use, if necessary.

### CODE REVIEW

**Comment by mbj36** [comment]
Thanks for the PR, It will be reviewed Shortly!

This message was posted by [GitMate.io](https://gitmate.io) with timestamp signature `Thanks for the PR, It will be reviewed Shortly!:1ePv80:7joNCDXd1nLXKCppzgBhy3jwbkw`

**Comment by mbj36** [comment]
CC - @gkrishnan724

**Comment by aks681** [comment]
@gkrishnan724 , can you please take a look at this and see if I have to change anything?
Thanks.

**Comment by gkrishnan724** [comment]
@aks681, I just suggest you to set the `maxDate` as null in datepicker.js. Also, remove your changes. Keeping the max-date as null and removing the `max="restrictedDate'` on `createfixeddepositproduct.html` on the input field should fix it.

**Comment by aks681** [comment]
@gkrishnan724 , yes, that too worked, made the change.

File: app/scripts/modules/datepicker.js [hunk 1]
```diff
@@ -109,7 +109,7 @@ angular.module('modified.datepicker', ['strap.position'])
             yearRange = getValue($attrs.yearRange, dtConfig.yearRange);
 
         this.minDate = dtConfig.minDate ? new Date(dtConfig.minDate) : null;
-        this.maxDate = dtConfig.maxDate ? new Date(dtConfig.maxDate) : null;
+        this.maxDate = null;
```
Comment (gkrishnan724) [inline app/scripts/modules/datepicker.js]: You are not supposed to make this null, please make the maxDate declaration in the config section null:
https://github.com/aks681/community-app/blob/4fed2e5d75d65aa3d65e93a55f884c60f4b479ad/app/scripts/modules/datepicker.js/#L96

**Comment by aks681** [comment]
@gkrishnan724 , changed it.

**Pull request merged**